renderer/index.js

```diff
@@ -595,6 +597,10 @@ function addTorrentEvents (torrent) {
     var torrentSummary = getTorrentSummary(torrent.infoHash)
     torrentSummary.status = 'seeding'
     updateTorrentProgress()
+    torrent.getFileModtimes(function (err, fileModtimes) {
+      if (err) return onError(err)
+      torrentSummary.fileModtimes = fileModtimes
```

Comment:
I made sure `saveState` works on app exit.

(If we were to save `config.json` every time `state.saved` changes, we might save it a bunch of times in succession.)